_posts/2016-05-20-drupalcon-nola-recap.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Drupalcon New Orleans 2016: a Twitter recap"
+date: 2016-05-20
+author: Chris Russo
+tags: drupal drupal8 drupalcon event
+summary: The ins and outs of the talks, sessions, parties, and more from Drupalcon New Orleans 2016.
+---
+
+<img src="/assets/img/blog/drupalcon-nola.png" alt="Drupalcon New Orleans logo">
+
+A week full of summits, sessions, a presentation, some parties and a fair amount of tweeting <img src="/assets/icons/twitter-bird.svg" height="24" alt="Twitter bird"> was one for the books at Drupalcon NOLA 2016. Come along for a Twitter-assisted chronological journey for Savas Labs in New Orleans.
+
+### Saturday / Sunday: Arrive, play soccer, get acquainted
+
+I arrived Saturday evening after a few short flights from Durham, NC. My gracious host and boots on the ground in NOLA Kevin Harrison lent me a bicycle, showed me around the city, and invited me to play in his soccer game. It was a comforting entry into a new city that closely resembled what my weekend would have been like if I were at home. Thanks a ton Kevin for the hospitality!
+
+ <img src="/assets/img/blog/kevin-harrison.jpg" alt="Kevin at a rooftop bar">
+ <span class="caption">See a palindrome anywhere?</span>
+
+ Kevin also took me to the convention center so I could get my bearings for the week ahead. He took my picture, and **_every so slightly_** doctored up the version you see below for me to share with my community that we had made it!
```

Comment:
Also "share with my community that we had made it" sounds a little weird to me - perhaps "our community" or "the Drupal community"?